lms/static/js/student_account/views/AccessView.js

```diff
@@ -1,280 +1,283 @@
+
+                    // Listen for 'auth-complete' event so we can enroll/redirect the user appropriately.
+                    this.listenTo( this.subview.login, 'auth-complete', this.authComplete );
+
+                },
+
+                reset: function( data ) {
+                    this.resetModel.ajaxType = data.method;
+                    this.resetModel.urlRoot = data.submit_url;
+
+                    this.subview.passwordHelp = new PasswordResetView({
+                        fields: data.fields,
+                        model: this.resetModel
+                    });
+
+                    // Listen for 'password-email-sent' event to toggle sub-views
+                    this.listenTo( this.subview.passwordHelp, 'password-email-sent', this.passwordEmailSent );
+
+                    // Focus on the form
+                    $('.password-reset-form').focus();
+                },
+
+                register: function( data ) {
+                    var model = new RegisterModel({}, {
+                        method: data.method,
+                        url: data.submit_url
+                    });
+
+                    this.subview.register =  new RegisterView({
+                        fields: data.fields,
+                        model: model,
+                        thirdPartyAuth: this.thirdPartyAuth,
+                        platformName: this.platformName
+                    });
+
+                    // Listen for 'auth-complete' event so we can enroll/redirect the user appropriately.
+                    this.listenTo( this.subview.register, 'auth-complete', this.authComplete );
+                },
+
+                institution_login: function ( unused ) {
+                    this.subview.institutionLogin =  new InstitutionLoginView({
+                        thirdPartyAuth: this.thirdPartyAuth,
+                        platformName: this.platformName,
+                        mode: this.activeForm
+                    });
+
+                    this.subview.institutionLogin.render();
+                },
+
+                hinted_login: function ( unused ) {
+                    this.subview.hintedLogin =  new HintedLoginView({
+                        thirdPartyAuth: this.thirdPartyAuth,
+                        hintedProvider: this.thirdPartyAuthHint,
+                        platformName: this.platformName
+                    });
+
+                    this.subview.hintedLogin.render();
+                }
+            },
 
-                // Listen for 'auth-complete' event so we can enroll/redirect the user appropriately.
-                this.listenTo( this.subview.register, 'auth-complete', this.authComplete );
+            passwordEmailSent: function() {
+                this.element.hide( $(this.el).find('#password-reset-anchor') );
+                this.element.show( $('#login-anchor') );
+                this.element.scrollTop( $('#login-anchor') );
```

Comment:
The #login-anchor jQuery element is being created twice, it would be better to make it a variable so you are just creating the object once and using it twice.